### PR TITLE
Admonitions for GitHub tip syntax.

### PR DESCRIPTION
**Pull request by ewels**
Closes nf-core/website#2409

### CODE REVIEW

**Review by mashehu — APPROVED** [review]

**Comment by mashehu** [comment]
what about documenting this? Something like:

````

## GitHub style admonitions

For Readme files you can also use the [GitHub-style admonitions/alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts):

```md
> [!NOTE]
> A magician wandered along the beach, but no one needed him.
```
````

**Comment by ewels** [comment]
On it 👍🏻 

> [!NOTE]
> A magician wandered along the beach, but no one needed him.

File: public/images/contributing/github_admonition_syntax.png
Comment (mashehu) [inline public/images/contributing/github_admonition_syntax.png:1]: need also a dark mode version of this 😉

Comment (mashehu) [reply public/images/contributing/github_admonition_syntax.png:1]: but tbh the link to the docs should be okay, no?

Comment (ewels) [reply public/images/contributing/github_admonition_syntax.png:1]: You think people will follow links to read docs? 😆 

I replaced with the stock image from github.com and made it smaller. Looks a bit better now imho.

Forgot that we could have dark mode / light mode images in the markdown. Not sure I can be bothered now though!

**Review by mashehu — APPROVED** [review]

File: src/content/docs/contributing/website_markdown.md [hunk 1]
```diff
@@ -131,21 +131,23 @@ Many people have speculated that if we knew exactly why the bowl of petunias had
...
 ## GitHub style admonitions
 
-For `README.md` files, you can also use the [GitHub style admonitions/alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts):
+For `README.md` files, you can also use the [GitHub style admonitions/alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).
+This _only_ works with markdown files fetched from pipeline repositories, so don't use it in markdown files in the website repo.
 
 ```md
 > [!NOTE]
-> A magician wandered along the beach, but no one needed him.
+> Useful information that users should know, even when skimming content.
```
[the remark this answers is not in the document]
Comment (ewels) [reply src/content/docs/contributing/website_markdown.md:139]: hah, it's really true - no one needed him! 🧙🏻

**Pull request merged**